apps/reducers/lib/services/stats.ex

```diff
@@ -30,8 +30,9 @@ defmodule Service.Stats do
     data = event.meta["data"]
     store_num = event.meta["store"]
     type = String.to_existing_atom(event.type)
-    preped_data = data_prep(data, type)
-    apply(__MODULE__, type, [model, preped_data])
+    # preped_data = data_prep(data, type)
+    # apply(__MODULE__, type, [model, preped_data])
```

Comment:
no, when moving things around I wanted to make sure i kept what was needed, obviously that isn't.